examples/http-server/main_test.go

```diff
@@ -20,9 +21,14 @@ import (
 	"gofr.dev/pkg/gofr/testutil"
 )
 
-const host = "http://localhost:9000"
+var host string
```

Comment:
Can we refactor to remove this variable?

I have no idea how difficult it could be. I'm only spotting it by looking at the partial diff